dev-build/app/Global.scala

```diff
@@ -69,6 +69,7 @@ with ConfigAgentLifecycle
 with SurgingContentAgentLifecycle
 with SectionsLookUpLifecycle
 with ABTHeadlinesLifecycle
+with MostPopularFacebookAutoRefresh.Lifecycle
```

Comment:
Ok, I hope that does get used in another pr, otherwise we've got another abstraction pattern for one.